bigchaindb/commands/utils.py

```diff
@@ -130,6 +158,10 @@ def mongodb_host(host):
                          help='Specify the location of the configuration file '
                               '(use "-" for stdout)')
 
+base_parser.add_argument('-l', '--log-level',
+                         choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
```

Comment:
@ttmc comment moved over to https://github.com/bigchaindb/bigchaindb/pull/1307/files#r106944637